### ISSUE TITLE
Glub Glub can't download WACVR

### ISSUE THREAD

**Issue by SageRhaposdy** [xiaopeng12138/WACVR#65]
Getting a Exception calling "DownloadFile" with "2" argument(s): "The remote server returned an error: (404) Not Found."  error when running Glub Glub.

**Comment by xiaopeng12138** [comment]
Idk if glub glub is using release link or something else. And idk which file you cant download. 
Can you maybe take a screen shot?

**Comment by SageRhaposdy** [comment]
Seems like its trying to download WACVR from https://nightly.link/xiaopeng12138/WACVR/workflows/build/main/artifact.zip which 404s.

I've bypassed the issue for now by downloading the release here and extracting the files to WACCA-Reverse-GlubGlub\WACVR manually.

**Comment by xiaopeng12138** [comment]
> Seems like its trying to download WACVR from https://nightly.link/xiaopeng12138/WACVR/workflows/build/main/artifact.zip which 404s.
> 
> I've bypassed the issue for now by downloading the release here and extracting the files to WACCA-Reverse-GlubGlub\WACVR manually. 

this link should be abandoned. idk if you can skip it or not. maybe you can try replace this link with the release link.

**Comment by SageRhaposdy** [comment]
Yep looks like both options work. If you extract the WaccaVR docs yourself to the folder, the GlubGlub script skips the step as well.
